src/consensus/block_body_validation.py

```diff
@@ -104,11 +104,13 @@ async def validate_block_body(
             prev_transaction_block.height,
```

Comment:
I guess `virtual_farmed_height` could be created here.

```
network_offset = int.from_bytes(self.constants.GENESIS_CHALLENGE[:16], "big") << 128
farmed_height = prev_transaction_block.height + network_offset
```
and pass that in instead of `prev_transaction_block.height`. You wouldn't have to pass in `GENESIS_CHALLENGE` that way, even fewer signatures would change, and we'd still get the height in the coin reward.

Reply:
Why don't we concatenate the clvm encoding of the height with the whole genesis challenge and hash that?